static/app/views/issueList/actions/index.spec.jsx

```diff
@@ -233,10 +235,57 @@ describe('IssueListActions', function () {
             },
           })
         );
+
+        expect(analyticsSpy).toHaveBeenCalledWith(
+          'issues_stream.archived',
+          expect.objectContaining({
+            status_details: {
+              ignoreUserCount: 300,
+              ignoreUserWindow: 10080,
+            },
+          })
+        );
       });
     });
   });
 
+  it('can archive an issue until escalating', async () => {
+    const analyticsSpy = jest.spyOn(analytics, 'trackAnalytics');
+    const org_escalating = {...organization, features: ['escalating-issues-ui']};
+    const apiMock = MockApiClient.addMockResponse({
+      url: `/organizations/${org_escalating.slug}/issues/`,
+      method: 'PUT',
+    });
+    jest.spyOn(SelectedGroupStore, 'getSelectedIds').mockReturnValue(new Set(['1']));
+
+    render(<WrappedComponent {...defaultProps} />, {organization: org_escalating});
+
+    await userEvent.click(screen.getByRole('button', {name: 'Archive'}));
+
+    expect(apiMock).toHaveBeenCalledWith(
+      expect.anything(),
+      expect.objectContaining({
+        query: {
+          id: ['1'],
+          project: [1],
+        },
+        data: {
+          status: 'ignored',
+          statusDetails: {},
+          substatus: 'until_escalating',
```

Comment:
can you update your branch i changed this string, sorry

Reply:
should be archived_until_escalating

Reply:
nice, thanks for letting me know!